plugins/fclrave/fclspace.cpp

```diff
@@ -152,10 +150,11 @@ void FCLSpace::ReloadKinBodyLinks(KinBodyConstPtr pbody, FCLKinBodyInfoPtr pinfo
                     enclosingBV += ConvertAABBToFcl(_tmpgeometry.ComputeAABB(Transform()));
                 }
             }
+            linkinfo->bFromExtraGeometries = false;
         }
 
         if( linkinfo->vgeoms.size() == 0 ) {
-            RAVELOG_DEBUG_FORMAT("env=%s, Initializing body '%s' (index=%d) link '%s' with 0 geometries (env %d) (userdatakey %s)", _penv->GetNameId()%pbody->GetName()%pbody->GetEnvironmentBodyIndex()%plink->GetName()%_penv->GetId()%_userdatakey);
+            RAVELOG_DEBUG_FORMAT("env=%s, Initializing body '%s' (index=%d) link '%s' with 0 geometries (env %d) (userdatakey %s, group=%s, bodygroup=%s)", _penv->GetNameId()%pbody->GetName()%pbody->GetEnvironmentBodyIndex()%plink->GetName()%_penv->GetId()%_userdatakey % _geometrygroup % pinfo->_geometrygroup);
```

Comment:
No spaces around `%` in format args, you can use this pre-commit hook to do it automatically

```bash
#!/bin/bash
set -eEuo pipefail

# Get the set of all files changed in the index, excluding deletions
git diff-index --name-only --cached --diff-filter=d -z HEAD |
    grep -E -z '\.cpp$|\.h$' |
    while read -r -d '' fn
    do
        # Get the diff for that file
        DIFF=`git diff --cached HEAD -- "$fn"`

        # Get the _complete_ un-diff as well
        REVERSEDIFF=`git diff -R HEAD -- "$fn"`

        # For all additions, ensure there are no spaces around percent
        NEWDIFF=`echo "$DIFF" | sed '/^+/s/ % /%/g'`

        # Take the original file to a base state
        TEMPFILE=`mktemp`
        echo "$REVERSEDIFF" | patch "$fn" -o "$TEMPFILE"

        # Reapply the modified staged change
        NEWFILE=`echo "$NEWDIFF" | patch "$TEMPFILE" -o -`

        # Patch index
        blob_sha1=$(echo "$NEWFILE" |
                        git hash-object -w --stdin)
        printf '100644 blob %s\t%s\0' "$blob_sha1" "$fn" |
            git update-index -z --index-info

        # Clean up the tempfile
        rm -- "$TEMPFILE"
    done || true
```

Reply:
thanks!
Fixed it!